engine/src/cmd/unit_generic.cpp

```diff
@@ -1493,81 +1081,11 @@ const std::vector< std::string >& Unit::GetDestinations() const
     return pImage->destination;
 }
 
-float Unit::TrackingGuns( bool &missilelock )
-{
-    float trackingcone = 0;
-    missilelock = false;
-    for (int i = 0; i < GetNumMounts(); ++i) {
-        if ( mounts[i].status == Mount::ACTIVE && (mounts[i].size&weapon_info::AUTOTRACKING) )
-            trackingcone = computer.radar.trackingcone;
-        if (mounts[i].status == Mount::ACTIVE && mounts[i].type->LockTime > 0 && mounts[i].time_to_lock <= 0)
-            missilelock = true;
-    }
-    return trackingcone;
-}
-
-void Unit::setAverageGunSpeed()
-{
-    float mrange = -1;
-    float grange = -1;
-    float speed  = -1;
-    bool  beam   = true;
-    if ( GetNumMounts() ) {
-        grange = 0;
-        speed  = 0;
-        mrange = 0;
-        int nummt = 0;
-        //this breaks the name, but... it _is_ more useful.
-        for (int i = 0; i < GetNumMounts(); ++i)
-            if (mounts[i].status == Mount::ACTIVE || mounts[i].status == Mount::INACTIVE) {
-                if ( mounts[i].type->isMissile() ) {
-                        if (mounts[i].type->Range > mrange)
-                            mrange = mounts[i].type->Range;
-                }
-                else {
-                    if (mounts[i].type->Range > grange)
-                        grange = mounts[i].type->Range;
-                    if (mounts[i].status == Mount::ACTIVE) {
-                        speed += mounts[i].type->Speed;
-                        ++nummt;
-                        beam  &= (mounts[i].type->type == weapon_info::BEAM);
-                    }
-                }
-            }
-        if (nummt) {
-            if (beam)
-                speed = FLT_MAX;
-            else
-                speed = speed/nummt;
-        }
-    }
-    this->missilerange = mrange;
-    this->gunrange     = grange;
-    this->gunspeed     = speed;
-}
-
-QVector Unit::PositionITTS( const QVector &absposit, Vector velocity, float speed, bool steady_itts ) const
-{
-    if (speed == FLT_MAX)
-        return this->Position();
-    float difficultyscale = 1;
-    if (g_game.difficulty < .99)
-        GetVelocityDifficultyMult( difficultyscale );
-    velocity = (cumulative_velocity.Scale( difficultyscale )-velocity);
-    QVector posit( this->Position()-absposit );
-    QVector curguess( posit );
-    for (unsigned int i = 0; i < 3; ++i) {
-        float time = 0;
-        if (speed > 0.001)
-            time = curguess.Magnitude()/speed;
-        if (steady_itts)
-            //** jay
-            curguess = posit+GetVelocity().Cast().Scale( time );
-        else
-            curguess = posit+velocity.Scale( time ).Cast();
-    }
-    return curguess+absposit;
-}
+
+
```

Comment:
there should only be 1-2 blank lines between functions

Reply:
Done.